app/Resources/views/page.html.twig

```diff
@@ -2,6 +2,10 @@
 
 {% block head %}
 
+    <script>
+        document.querySelector('html').classList.add('js');
```

Comment:
no need to tie into browser events to wait for?

Reply:
No, we want this in place as early as possible.